### DISCUSSION TITLE
LAN connections timing out

### DISCUSSION THREAD

**Discussion by Kick4U2** [naim94a/udpt#130] [Q&A]
Hello, I'm eager to get this working, but it seems that I can't get my clients to connect to the tracker.
I tried binding my UDP address to 0.0.0.0, 192.168.0.215. and my WAN IP but it's not working.

It looks like I need to enable "allow_iana_ips" in the configuration, but the Wiki here on GitHub.com is not in line with the documentation on GitHub.io.

What does it take to allow this tracker to work with both LAN and WAN clients?

**Comment by naim94a** [comment]
The current behavior is to allow all ips; 
There was a bug in the tracker that was recently fixed and may have fixed this problem

**Reply by exalented** [reply]
I've just built this and I can't seem to get this to work either.

**Reply by naim94a** [reply]
Hmm, could you please describe your setup?

**Reply by exalented** [reply]
Ahh, so I've found something. I'm able to get clients to show that the tracker is working, but nothing past that. udpt is responding with the ip of the running container instead of the server's actual ip.:
![image](https://user-images.githubusercontent.com/9328864/151695668-a7610cd2-0146-4ff5-a571-253f5a16822b.png)

To answer your question, I've written a docker-compose.yml to build the Dockerfile, open ports and changed the ports and token set in udpt.toml. My torrent clients are rtorrent and qbittorrent.
